samtranslator/model/eventsources/push.py

```diff
@@ -607,6 +610,59 @@ def _add_swagger_integration(self, api, function):
                 editor.add_request_model_to_method(path=self.Path, method_name=self.Method,
                                                    request_model=self.RequestModel)
 
+        if self.RequestParameters:
+
+            default_value = {
+                'Required': False,
+                'Caching': False
+            }
+
+            parameters = []
+            for parameter in self.RequestParameters:
+
+                if isinstance(parameter, dict):
+
+                    parameter_name, parameter_value = next(iter(parameter.items()))
+
+                    if not re.match('method\.request\.(query|path|header)', parameter_name):
```

Comment:
see comments above about the swagger spec.  It should be `query` and not `querystring` even though the Amazon extension uses `querystring` instead of `query`.  Testing also verified this.

Reply:
I've updated this to use `querystring`.